nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/FacetDescriptor.java

```diff
@@ -34,6 +36,9 @@
     @XNode("@perDocumentQuery")
     public Boolean perDocumentQuery;
 
+    @XNode("@enabled")
+    public Boolean enabled;
```

Comment:
Why not using `public boolean enabled = true` here? As you never check if `enabled` is `null`, and it may make more sense to have enabled to `true` by default.

Reply:
I prefer to use this pattern in all descriptors (descriptors as pure data carriers), because it facilitates writing correct `merge` code. Although it's true that in this specific case the merge is done totally ad hoc in `SchemaManagerImpl` because there's interactions between several descriptors. But the pattern is worth keeping as it's not too much work.